### PR TITLE
Add remaining python3 modules

### PR DESCRIPTION
**Pull request by n8henrie**
I've noted a couple missing packages, so today I wanted to see if I could get the remainder. I have the html version of the 3.6.3 docs downloaded locally for quick reference, so I figured this would be an easy way to scrape most of the missing modules.

I wrote a quick script to import the scraped modules in a fresh venv to make sure all of these are importable and hopefully eliminate any false positives. I used a combination of the below strategies, all of which were run from `python-3.6.3-docs-html/library`

Collect the HTML filenames, which for the most part seem to be importable modules, and if they are importable and not in `stdlib.txt` (a text file of flake8-import-order's existing stdlib list), then accumulate them:
```shell_session
for htmlfile in *.html; do lib="${htmlfile%.html}"; if ! grep --quiet "${lib}" stdlib.txt; then python3 -c "import ${lib}" &> /dev/null && echo "${lib}" >> good_imports.txt; fi; done
```

Scrape the HTML files for any `<div>` with an `id` starting with `module-`:
```shell_session
$ grep -h -P -o '(?<=<div class="section" id="module-).*(?=">)' *.html | sort -u > all_modules.txt
```

Try to import each module; accumulate the ones that work and print the ones that result in an error.
```shell_session
$ for lib in $(cat all_modules.txt); do python3 -c "import ${lib}" &> /dev/null && echo "${lib}" >> good_imports.txt || echo "${lib}"; done
```

Then I just basically did a `sort --unique` with the existing stdlib list.

Anyway, may want to look it over to make sure I didn't screw something up, but I think that having tested for import-ability in a fresh venv should mean all the additions are valid.

### CODE REVIEW

File: flake8_import_order/stdlib_list.py [hunk 1]
```diff
@@ -112,6 +115,21 @@
...
+    "email.header",
+    "email.headerregistry",
+    "email.iterators",
+    "email.message",
+    "email.mime",
+    "email.parser",
+    "email.policy",
+    "email.utils",
+    "encodings.idna",
+    "encodings.utf_8_sig",
```
Comment (pgjones) [inline flake8_import_order/stdlib_list.py]: I think there is also `encodings.mbcs`

**Comment by pgjones** [comment]
I'm not sure it is necessary to include anything beyond the root package name, i.e. `email` is enough as `email.charset` is considered as email and hence stdlib.

File: flake8_import_order/stdlib_list.py (hunk 1 above)
Comment (pgjones) [reply flake8_import_order/stdlib_list.py]: Actually, no need to worry about the subpackages only the root is needed.

**Comment by n8henrie** [comment]
Looks like you're right -- I was confused because I had to add
`xmlrpc.client` and `xmlrpc.server` in
3f5906e1057a1792f4b6483b653740d99cc6ffd1 -- but just realized that I
probably just should have just added `xmlrpc` (which looks like it may have
been a py3 rename from `xmlrpclib`, which was already there).

**Comment by pgjones** [comment]
I think you are right and that `xmlrpc` would have been sufficient. Could you update to just be the root packages? Then I'll happily merge this valuable change.

**Comment by n8henrie** [comment]
There seem to be a lot of packages that are not in 2.7.14 or 3.6.3 -- Are these mostly relics from old python versions? Or maybe some are Windows only?

```shell_session
$ for lib in $(cat libs.txt); do venv/bin/python3 -c "import ${lib}" &> /dev/null ||
{ venv2/bin/python2 -c "import ${lib}" &> /dev/null || echo "${lib}"; }; done
AL
Binary
Boolean
DEVICE
EasyDialogs
FL
FrameWork
GL
Mapping
MiniAEFrame
Numeric
SUNAUDIODEV
Sequence
Set
Text
al
bsddb
cd
dbhash
dl
fl
flp
fm
fpectl
gensuitemodule
gl
imageop
imgfile
jpeg
msilib
ossaudiodev
spwd
sunaudiodev
winsound
```

**Comment by pgjones** [comment]
Thanks, it is a good question about those that are left. Some look to have been removed with Python 3 (EasyDialogs for example), but otherwise I don't know. I think this should be cleaned up when this project drops Python2.

**Pull request merged**